protocol/dydx-cosmwasm/src/serializable_int.rs

```diff
@@ -0,0 +1,74 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize, Deserializer, Serializer};
+use num_bigint::BigInt;
+use schemars::schema::{Schema, SchemaObject, InstanceType};
+use std::fmt;
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct SerializableInt {
+    i: BigInt,
+}
+
+impl SerializableInt {
+    pub fn new(i: BigInt) -> Self {
+        Self { i }
+    }
+    pub fn to_big_int(&self) -> &BigInt {
```

Comment:
```suggestion
    pub fn as_big_int(&self) -> &BigInt {
```
This is more idiomatic for reference conversions. See [example](https://doc.rust-lang.org/std/string/struct.String.html#method.as_str)